nbstripout/_utils.py

```diff
@@ -94,6 +95,25 @@ def strip_zeppelin_output(nb: dict) -> dict:
             cell['results'] = {}
     return nb
 
+def match_output_type(output: Dict,
+                      output_type: str) -> bool:
+    """
+    Take the `output_type` string, and return whether the output matches.
+
+    Currently, supported formats are `output_type:name` or `output_type`.
+
+    :param output: The output dictionary from a notebook cell.
+    :param output_type: User-provided string to match against.
+    """
+
+    # Check if the ':' format is used, and if so, split into output_type and name
+    name = None
+
+    if ':' in output_type:
+        output_type, name = re.search(r'^(.*?):(.*)$', output_type).groups()
```

Comment:
Can't you simply use split?
```suggestion
        output_type, name = output_type.split(':')
```